src/transformers/models/bigbird_pegasus/configuration_bigbird_pegasus.py

```diff
@@ -183,3 +187,32 @@ def __init__(
             decoder_start_token_id=decoder_start_token_id,
             **kwargs,
         )
+
+
+class BigBirdPegasusOnnxConfig(OnnxConfigWithPast):
+    @property
+    def inputs(self) -> Mapping[str, Mapping[int, str]]:
+        common_inputs = OrderedDict({"input_ids": {0: "batch", 1: "sequence"}})
+        if self.use_past:
+            for i in range(self._config.encoder_layers * 2):
+                common_inputs[f"past_key_values.{i}.key"] = {0: "batch", 2: "past_sequence"}
+                common_inputs[f"past_key_values.{i}.value"] = {0: "batch", 2: "past_sequence"}
+
+            common_inputs["attention_mask"] = {0: "batch", 1: "past_sequence + sequence"}
+        else:
+            common_inputs["attention_mask"] = {0: "batch", 1: "sequence"}
+        # common_inputs["labels"] = {0: "batch", 1: "sequence"}
+        return common_inputs
+
+    @property
+    def outputs(self) -> Mapping[str, Mapping[int, str]]:
```

Comment:
Isn't it task dependent?
You can actually call `super().outputs`, which wil return the proper outputs for the task at hand, and then add the outputs related to the `past_key_values`.